test/github/client/installation-token-cache.test.ts

```diff
@@ -1,5 +1,6 @@
 import InstallationTokenCache from "../../../src/github/client/installation-token-cache";
 import AuthToken, { ONE_MINUTE, TEN_MINUTES } from "../../../src/github/client/auth-token";
+jest.unmock("lru-cache");
```

Comment:
we mock it globally in `setup.ts` so that we have to be explicit in our nocks what is being called.  This removes a lot of flakiness when it comes to running all tests vs a single test, or if the tests are not run in a specific order.

Reply:
in this case though, since we're trying to test the caching itself, we unmock it so lru-cache works as it should.